### PR TITLE
add package cewe fotobuch

### PR DESCRIPTION
**Pull request by fkz**

### CODE REVIEW

File: pkgs/applications/graphics/cewe-fotobuch/EULA.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿Lizenz- und Nutzungsbedingungen „Mein CEWE FOTOBUCH“
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/EULA.txt]: What is that file?

File: pkgs/applications/graphics/cewe-fotobuch/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, libtool, libjpeg, xz, xlibs, zlib, mesa, glib, gstreamer, libxml2, cups, sqlite, expat,
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/default.nix]: Add line breaks to all that file.

File: pkgs/applications/graphics/cewe-fotobuch/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, libtool, libjpeg, xz, xlibs, zlib, mesa, glib, gstreamer, libxml2, cups, sqlite, expat, 
+ gst_plugins_base, qt5, fontconfig, freetype, which, cdrkit, makeWrapper, perl, wget, ncurses, xdg_utils, unzip, less}:
+
+let libtiff = import ./libtiff4.nix { inherit stdenv fetchurl zlib libjpeg; };
+    rpath = stdenv.lib.makeSearchPath "lib" [ stdenv.glibc libtool
+     xlibs.libX11 xlibs.libXrender xlibs.libSM xlibs.libICE xlibs.libxcb cups zlib mesa glib gstreamer gst_plugins_base libxml2
+     sqlite expat qt5.base qt5.tools qt5.svg qt5.declarative qt5.graphicaleffects qt5.quickcontrols qt5.webkit libtiff fontconfig freetype] + 
+    ":${stdenv.cc.cc}/lib64"; in
+stdenv.mkDerivation {
+   name = "cewe-fotobuch";
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/default.nix]: Add version.

File: pkgs/applications/graphics/cewe-fotobuch/default.nix [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      for file in $(find $out -type f); do
+	patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$file" || true
+        patchelf --set-rpath ${rpath}:\$ORIGIN "$file" || true
+      done
+      makeWrapper "$out/Mein\ CEWE\ FOTOBUCH" $out/bin/cewe-fotobuch \
+	--prefix PATH : ${which}/bin:${cdrkit}/bin \
+	--prefix QT_QPA_PLATFORM_PLUGIN_PATH : ${qt5.base}/lib/qt5/plugins
+     '';
+  meta = with stdenv.lib; {
+    description = "CEWE Fotobuch is a german software to create and order photo books";
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/default.nix]: Omit pkg name.

File: pkgs/applications/graphics/cewe-fotobuch/install.pl [hunk 5]
```diff
@@ -0,0 +1,973 @@
+#! /usr/bin/perl -w
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/install.pl]: Why do we need that file?

File: pkgs/applications/graphics/cewe-fotobuch/libtiff4.nix [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  };
+
+  propagatedBuildInputs = [ zlib libjpeg ];
+
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "Library and utilities for working with the TIFF image file format";
+    homepage = http://www.libtiff.org/;
+    license = "bsd";
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/libtiff4.nix]: Use `stdenv.lib.licenses`.

File: pkgs/applications/graphics/cewe-fotobuch/libtiff4.nix [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, zlib, libjpeg }:
+
+let version = "3.9.7"; in
+
+stdenv.mkDerivation rec {
+  name = "libtiff-${version}";
```
Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/libtiff4.nix]: Btw, we already have `libtiff` in `nixpkgs`.

**Comment by jagajaga** [comment]
Don't forget to squash commits.

File: pkgs/applications/graphics/cewe-fotobuch/EULA.txt (hunk 1 above)
Comment (the-kenny) [inline pkgs/applications/graphics/cewe-fotobuch/EULA.txt]: Terms and conditions in German.

Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/EULA.txt]: I see, but why do we need them?

Comment (the-kenny) [inline pkgs/applications/graphics/cewe-fotobuch/EULA.txt]: It's an unfree package, so I suppose we should somehow reference the terms&conditions. For DwarfFortress, we ship a copy of an email allowing us to distribute it too.

Comment (jagajaga) [inline pkgs/applications/graphics/cewe-fotobuch/EULA.txt]: Oh, ok. But may be there is an English version?

File: pkgs/applications/graphics/cewe-fotobuch/install.pl (hunk 5 above)
Comment (the-kenny) [inline pkgs/applications/graphics/cewe-fotobuch/install.pl]: It's actually the installer.

Full stop: This file downloads the actual installation files without ```fetchurl``` etc. This won't work with chrooted builds. 

This needs to be fixed before we can merge this.

**Comment by pSub** [comment]
@fkz any updates?

**Comment by fkz** [comment]
I'm closing this now since I'm not really interested in merging this considering expected work vs benefit.

**Pull request closed without merging**